sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,17 +1,24 @@
 # Release History
 
-## 2.5.0-beta.1 (Unreleased)
+## 2.5.0 (2022-05-06)
 
 ### Features Added
 
-### Breaking Changes
+- Added `ProxyOptions.fromConfiguration(Configuration)` to enable creation of `ProxyOptions` from an environment 
+  configuration.
 
 ### Bugs Fixed
+
 - Fixed proxy authentication type not being read from configuration. ([#28073](https://github.com/Azure/azure-sdk-for-java/issues/28073))
 - Updated ProxyOptions.SYSTEM_DEFAULTS to use ProxyAuthenticationType.NONE
 
 ### Other Changes
 
+#### Dependency Updates
+
+- Upgraded `azure-core` from `1.27.0` to `1.28.0`.
+- Upgraded Reactor from `3.4.14` to `3.4.17`.
```

Comment:
This should be called out in `azure-core` changelog. Core amqp doesn't have a direct dependency on Reactor.